python/cudf/dataframe/string.py

```diff
@@ -626,6 +626,26 @@ def copy(self, deep=True):
     def unordered_compare(self, cmpop, rhs):
         return string_column_binop(self, rhs, op=cmpop)
 
+    def find_and_replace(self, to_replace, replacement, all_nan):
+        """
+        Return col with *to_replace* replaced with *value*
+        """
+        to_replace = columnops.as_column(to_replace)
```

Comment:
Any reason we're converting to a column here just to copy it back to host later?

Reply:
Basically in anticipation of `to_replace` and `replacement` being something other than a string, e.g., series or list of size 1